english/named_entity_recognition.md

```diff
@@ -1,15 +1,30 @@
 # Named entity recognition
 
+1. [Intro](#intro)
+1. [Datasets](#datasets)
+ 1. [CoNLL 2003 (English)](#conll-2003-english)
+ 1. [Long-tail emerging entities](#long-tail-emerging-entities)
+ 1. [Ontonotes v5 (English)](#ontonotes-v5-english)
+
+ 
+
+## Intro 
+
 Named entity recognition (NER) is the task of tagging entities in text with their corresponding type.
-Approaches typically use BIO notation, which differentiates the beginning (B) and the inside (I) of entities.
-O is used for non-entity tokens.
+Approaches typically use BIO2 notation, which differentiates the beginning (B) and the inside (I) of entities.
```

Comment:
As far as I'm aware, BIO notation is the standard so probably better to revert this. What is the difference between BIO and BIO2?